### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by Jasha10**
This PR creates a `PULL_REQUEST_TEMPLATE.md` file nudging potential
contributors to (1) add tests to their PR and (2) check out the
`CONTRIBUTING.md` file in the repo's root.

The template is based on https://github.com/facebookresearch/hydra/blob/main/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

**Comment by omry** [comment]
Maybe add a fixes block to encourage people to have a corresponding issues that PR is addressing.

**Comment by Jasha10** [comment]
Updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Yes/No
+
+## Test Plan
+
+(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)
+
+## Fixes
+
+What issue does this PR fix? Use https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue to link this PR to a corresponding issue.
+
```
Comment (omry) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
What issue does this PR fix? Use https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue to link this PR to a corresponding issue.

# E.g 
# Fixes #12345
Fixes #<issue_number>
```

**Review by omry — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Fixes #<issue_number>
```
Comment (Jasha10) [inline .github/PULL_REQUEST_TEMPLATE.md:19]:
```suggestion
Fixes #<issue_number>

```

**Pull request merged**